llvm/lib/Target/AArch64/AArch64SVEInstrInfo.td

```diff
@@ -3887,7 +3887,7 @@ let Predicates = [HasSVE2orSME] in {
   defm WHILERW_PXX : sve2_int_while_rr<0b1, "whilerw", "int_aarch64_sve_whilerw">;
 } // End HasSVE2orSME
 
-let Predicates = [HasSVE2AES] in {
+let Predicates = [HasSVE2, HasSVEAES] in {
```

Comment:
So these instructions are now available when: 
HasSSVE_AES  or HasSVE2 and  HasSVEAES
 Because the instructions  should be able to encode/decode only with  +ssve_eas and atm  it will fail.

https://developer.arm.com/documentation/ddi0602/2024-09/SVE-Instructions/AESE--vectors---AES-single-round-encryption-?lang=en